src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let money = days * 40;
+
+  if (days >= 7) {
+    money = money - 50;
+  }
+
+  if (days >= 3 && days < 7) {
+    money = money - 20;
```

Comment:
Replace the literals '3', '7', and '20' with named constants to clarify their meaning. For example, 'const SHORT_TERM_RENTAL = 3;' and 'const SHORT_TERM_DISCOUNT = 20;'. Also, since the condition 'days < 7' is unnecessary due to the return statement in the previous condition, it can be removed.